assets/activation-scripts/activate

```diff
@@ -79,6 +79,7 @@ FLOX_CMD=""
 FLOX_TURBO="${FLOX_TURBO:-}"
 FLOX_NOPROFILE="${FLOX_NOPROFILE:-}"
 _FLOX_ENV_ACTIVATION_MODE="dev"
+_FLOX_WATCHDOG_BIN=""
```

Comment:
nonblocking: that doesn't seem like a sensible default - would it be better to error after the while if no `--watchdog` is provided?

Reply:
We use the absence of this value (both before and after this change) to _not_ start a watchdog for container and wrapped-binary invocations: https://github.com/flox/flox/blob/cc611ccc4c3a9855975ca3d0a02bd776816594ee/assets/activation-scripts/activate.d/start.bash#L55-L64

Reply:
Ah makes sense I'd forgotten